src/app/GitUI/UserControls/BlameControl.cs

```diff
@@ -82,6 +83,13 @@ public BlameControl()
             _gitBlameParser = new GitBlameParser(() => UICommands.Module);
         }
 
+        public void InitSplitterManager(SplitterManager splitterManager, string parentContainerPrefix)
+        {
+            string containerPrefix = $"{parentContainerPrefix}{Name}.";
+            splitterManager.AddSplitter(splitContainer1, $"{containerPrefix}{splitContainer1.Name}", defaultDistance: splitContainer1.Panel1MinSize + 1);
+            splitterManager.AddSplitter(splitContainer2, $"{containerPrefix}{splitContainer2.Name}", defaultDistance: splitContainer2.Panel1MinSize + 1);
```

Comment:
What is the clue behind `+ 1`?

Reply:
There is a validation in a `SplitterManager` where this has to be true. https://github.com/mdonatas/gitextensions/blob/remember-blame-splitter-pos/src/app/GitUI/SplitterManager.cs#L147-L150

I suppose the logic is that the splitter can't occupy the same position as `panel1` at it's smallest size as that would require it to get even smaller to leave space for the splitter and it can't.